### ISSUE TITLE
rST badge code should have alt text

### ISSUE THREAD

**Issue by ssokolow** [landscapeio/landscape-issues#11]
Currently, the reStructuredText badge code pastes like this.

``` rST
.. image:: https://landscape.io/github/ssokolow/snakebyte/master/landscape.png
:target: https://landscape.io/github/ssokolow/snakebyte/master
```

However, indent aside (which is an issue), since it's hyperlinked, alt text is especially important. I'd suggest "Code Metrics", resulting in this:

``` rST
.. image:: https://landscape.io/github/ssokolow/snakebyte/master/landscape.png
   :target: https://landscape.io/github/ssokolow/snakebyte/master
   :alt: Code Metrics
```

**Comment by carlio** [comment]
Fixed!
